src/newsletter-editor/debug-send/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { registerPlugin } from '@wordpress/plugins';
+import { format } from '@wordpress/date';
+
+function NewslettersDebugSend() {
+	const { sendErrors, sent } = useSelect( select => {
+		const { getCurrentPostAttribute } = select( 'core/editor' );
+		const meta = getCurrentPostAttribute( 'meta' );
+		return {
+			sendErrors: meta.newsletter_send_errors,
+			sent: meta.newsletter_sent,
+		};
+	} );
+	if ( sent || ! sendErrors?.length ) {
+		return null;
+	}
+	return (
+		<PluginDocumentSettingPanel
+			name="newsletters-ads-settings-panel"
+			title={ __( 'Debug Send Errors', 'newspack-newsletters' ) }
```

Comment:
Suggestion: 

```suggestion
			title={ __( 'Send Errors', 'newspack-newsletters' ) }
```

"debugging" feels too technical. But maybe it's just me.